src/environments/environment.prod.ts

```diff
@@ -1,4 +1,5 @@
 export const environment = {
   production: true,
-  graphqlUri: '/graphql'
+  graphqlUri: '/graphql',
+  appTitle: 'Hypertrace'
```

Comment:
if this is the source of truth now (which makes sense), let's remove the setting in index.html

Reply:
done @aaron-steinfeld